app.rb

```diff
@@ -23,8 +23,10 @@ class App < Sinatra::Base
     @bugzilla = Nailed::Config.content.fetch("bugzilla")
     @products = Nailed::Config.products
     @product_query = @products.join("&product=")
-    @orgs = Nailed::Config.organizations
-    @org_query = @orgs.map { |o| o.name.dup.prepend("user%3A") }.join("+")
+    @orgs = Nailed::Config.organizations['github']
```

Comment:
Is this fine, that @org_query only has GitHub organizations? It is used to generate a GitHub URL for a pull request in one of the index javascript functions, but wouldn't we need something similar for GitLab?

Reply:
You are absolutely right about that, but I couldn't find a similar feature to display all Merge requests in gitlab.

Reply:
@parlt91  Are you looking for something like` /merge_requests?scope=all&state=all`

Reply:
I am looking for something similar to e.g. https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Acrowbar+user%3Akubic-project
If that's what it does then, yes.

Reply:
@simson-suse Since there does not seem to be an analogon for gitlab.
I will leave it as it is.